src/__tests__/pasteTokenReauth.test.ts

```
@@ -0,0 +1,108 @@
+// URL embeds instanceId (as `apiKey.instanceId` or `?instanceId=`), so
+// recreating hands back a different URL and the connector has to be re-added
+// in Claude. OAuth providers get in-place repair via /connect/:slug?reconnect=;
+// this is the paste-token counterpart.
+
+const repoRoot = path.join(path.dirname(fileURLToPath(import.meta.url)), '..', '..');
+const dashboardPath = path.join(repoRoot, 'public', 'dashboard.html');
+const webServerPath = path.join(repoRoot, 'src', 'website', 'webServer.ts');
+
+function loadUsesPastedToken(): (instance: any, mcp: any) => boolean {
+  const html = fs.readFileSync(dashboardPath, 'utf8');
+  const match = html.match(/function usesPastedToken\(instance, mcp\) \{[\s\S]*?\n    \}/);
```

ESLint

[error] Spaces are hard to count. Use {4}.

(no-regex-spaces)